libraries/AP_NavEKF3/AP_NavEKF3_Outputs.cpp

```diff
@@ -451,6 +451,13 @@ void  NavEKF3_core::getVariances(float &velVar, float &posVar, float &hgtVar, Ve
     offset   = posResetNE;
 }
 
+// return the diagonals from the covariance matrix
+void  NavEKF3_core::getStateVariances(float stateVar[24])
```

Comment:
shouldn't stateVar be a pointer ? otherwith it will create the table on function call and destroy it on exit, so we return anything